drools-wb-screens/drools-wb-guided-template-editor/drools-wb-guided-template-editor-backend/src/test/java/org/drools/workbench/screens/guided/template/server/indexing/IndexGuidedRuleTemplateAttributesTest.java

```diff
@@ -49,6 +50,13 @@ public void testIndexGuidedRuleTemplateAttributes() throws IOException, Interrup
                                                                                           add(new Import("org.drools.workbench.screens.guided.template.server.indexing.classes.Mortgage"));
                                                                                       }},
                                                                                       "template1");
+
+
+        model.addAttribute(new RuleAttribute("salience",
```

Comment:
Should we also assert that there is no index entries for these attributes? 

Otherwise I don't see any value in adding these lines to the test.

Reply:
Right now they pick up if the test throws an exception, but I'll add assertion with comments saying these are not indexed.

Reply:
Yes that was what I was meaning with the "assertion". I'll do that.

Reply:
@manstis ok so can't search with "salience" since it is not a recognized PartType.